pkg/sandbox/teardown.go

```diff
@@ -33,6 +51,24 @@ func Teardown(ctx context.Context, cli docker.Docker) error {
 	return nil
 }
 
+func printTeardownLogs(ctx context.Context, cli docker.Docker) {
+	fmt.Print("\n---- Verbose Logs ----\n")
+	var data = os.Stdout
+	table := tablewriter.NewWriter(data)
+	table.SetHeader([]string{"ID", "From", "Type", "Action"})
+	table.SetRowLine(true)
+	table.ClearRows()
+	table.SetAutoWrapText(true)
+	table.SetAutoFormatHeaders(true)
+	msgs, _ := cli.Events(ctx, types.EventsOptions{
+		Since: "1m",
+	})
+	table.Render()
+	for m := range msgs {
+		table.RenderRowOnce([]string{m.ID, m.From, m.Type, m.Action})
+	}
+}
```

Comment:
@pmahindrakar-oss as this is a `chan` which is run by a goroutine, i have no other way to render upcoming messages (other than rendering the whole table again when new message comes, which will print multiple tables). Hence i was forced to use the `RenderRowOnce` function instead, the caveat being the title's won't adjust themselves to the size. 

I was not able to find a workaround for this, hence the UX issue. Any suggestions would be helpful

Reply:
You can use consume the message in the same go routine and  print in the main routine. you can use wait groups for this. And also would be preferable if you utilize the pkg/printer utility

Reply:
@DarthBenro008, would you be able to work on this?